plugins/gcpaudit/rules/gcp_auditlog_rules.yaml

```
@@ -20,7 +20,7 @@
 
 - required_plugin_versions:
   - name: gcpaudit
-    version: 0.3.0
+    version: 0.3.1
```

```suggestion
    version: 0.3.0
```

No need to bump this, since the ruleset is still compatible with plugin version 0.3.0